custom/icds_reports/tests/agg_tests/reports/test_bihar_api_mother_details.py

```diff
@@ -1,17 +1,19 @@
 import datetime
-from django.test import TestCase
-from custom.icds_reports.reports.bihar_api import get_mother_details
-from custom.icds_reports.tasks import  update_bihar_api_table
 from datetime import date
 
+from django.test import TestCase
 from mock import patch
 
+from custom.icds_reports.reports.bihar_api import get_mother_details
+from custom.icds_reports.tasks import update_bihar_api_table
+
 
-@patch('custom.icds_reports.utils.aggregation_helpers.distributed.bihar_api_demographics.BiharApiDemographicsHelper.bihar_state_id',
-       'st1')
+@patch(
+    'custom.icds_reports.utils.aggregation_helpers.distributed.bihar_api_demographics.BiharApiDemographicsHelper.bihar_state_id',
+    'st1')
 class BiharAPIMotherTest(TestCase):
 
-    def test_file_content(self):
+    def test_file_content_1(self):
```

Comment:
it would be good to give these more specific names that 1 and 2, so its clear what each case tests

Reply:
This isnt really testing "file content" either, for that matter